.github/CODEOWNERS

```diff
@@ -139,8 +139,7 @@ core/scripts/gateway @smartcontractkit/dev-services
 /integration-tests/**/*ccip* @smartcontractkit/ccip-offchain @smartcontractkit/core
 
 # Deployment tooling
-# Initially the common structures owned by CCIP
-/deployment @smartcontractkit/ccip @smartcontractkit/keystone @smartcontractkit/core
+/deployment @smartcontractkit/ccip @smartcontractkit/keystone @smartcontractkit/core @smartcontractkit/deployment-automation
```

Comment:
Ooh, I was going to suggest deployment-automation should also be on sub-packages of /deployment, for purposes of refactors and infrastructure improvements.